liblangutil/Scanner.cpp

```diff
@@ -143,10 +141,10 @@ class LiteralScope
 }; // end of LiteralScope class
 
 
-void Scanner::reset(CharStream const& _source, string const& _sourceName)
+void Scanner::reset(CharStream _source, string _sourceName)
 {
-	m_source = _source;
-	m_sourceName = make_shared<string const>(_sourceName);
+	m_source = std::move(_source);
+	m_sourceName = make_shared<string const>(std::move(_sourceName));
```

Comment:
I think this changeset would have deserved its own commit as it has nothing to do with namespace changes.

Reply:
i agree. (next time) :-)